starter-code/src/app/app.component.html

```diff
@@ -1,3 +1,4 @@
-<h1>
-  {{title}}
-</h1>
+
+  <app-log-form-component></app-log-form-component>
```

Comment:
this should be the router outlet and then you need a page where youll put the form 😄